.gitignore

```diff
@@ -21,3 +21,13 @@
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+
+#Ballerina configuartion file
+ballerina.conf
+
+#Target folder of ballerina project
+/target
+
+#json files created inside resources folder
+/src/azure_cosmosdb/resources/*.json
+
```

Comment:
Add ballerina-internal.log.lck file to gitignore

Reply:
This implementation is in SLP4 therefore lock file is included in the gitignore. Once the connector is migrated to SLP7 it will be removed.